cmd/crossplane/core/core.go

```diff
@@ -114,20 +114,23 @@ type startCommand struct {
 	EnableRealtimeCompositions bool `group:"Alpha Features:" help:"Enable support for realtime compositions, i.e. watching composed resources and reconciling compositions immediately when any of the composed resources is updated."`
 	EnableSSAClaims            bool `group:"Alpha Features:" help:"Enable support for using Kubernetes server-side apply to sync claims with composite resources (XRs)."`
 
-	EnableCompositionFunctions               bool `default:"true" group:"Beta Features:" help:"Enable support for Composition Functions."`
-	EnableCompositionFunctionsExtraResources bool `default:"true" group:"Beta Features:" help:"Enable support for Composition Functions Extra Resources. Only respected if --enable-composition-functions is set to true."`
 	EnableCompositionWebhookSchemaValidation bool `default:"true" group:"Beta Features:" help:"Enable support for Composition validation using schemas."`
 	EnableDeploymentRuntimeConfigs           bool `default:"true" group:"Beta Features:" help:"Enable support for Deployment Runtime Configs."`
 
 	// These are GA features that previously had alpha or beta feature flags.
 	// You can't turn off a GA feature. We maintain the flags to avoid breaking
 	// folks who are passing them, but they do nothing. The flags are hidden so
 	// they don't show up in the help output.
-	EnableCompositionRevisions bool `default:"true" hidden:""`
+	EnableCompositionRevisions               bool `default:"true" hidden:""`
+	EnableCompositionFunctions               bool `default:"true" hidden:""`
+	EnableCompositionFunctionsExtraResources bool `default:"true" hidden:""`
```

Comment:
Maybe a thought but have we given enough time for Extra Resources since this came out I believe in 1.16? I guess we put it behind the same feature flag as composition functions as a whole but maybe food for thoughts